### PR TITLE
fix issue #115

### PR DESCRIPTION
**Pull request by caosiyang**
support to run benchmark on mongodb with authentication disabled

### CODE REVIEW

**Comment by siddontang** [comment]
thanks @caosiyang 

please sign the CLA

**Comment by caosiyang** [comment]
> thanks @caosiyang
> 
> please sign the CLA

done

File: README.md [hunk 1]
```diff
@@ -220,10 +220,10 @@ Common configurations:
 |field|default value|description|
 |-|-|-|
 |mongodb.uri|"mongodb://localhost:27017"|Connection string URI format|
-|mongodb.ns|"ycsb.ycsb"|Namespace|
-|mongodb.user|"root"|Database Username|
-|mongodb.passw|""|Database Password|
-|mongodb.dbauth|"admin"|Admin Database|
+|mongodb.namespace|"ycsb.ycsb"|Namespace to use|
+|mongodb.authenticationDatabase|"admin"|Authentication database|
```
Comment (siddontang) [inline README.md]: the name is too long, can you shorten it?

Comment (caosiyang) [reply README.md]: The name "authenticationDatabase" refers to "mongo" command options.
Is "authdb" ok?

Comment (caosiyang) [reply README.md]: "mongo --help" output like this:
```
Authentication Options:
  -u [ --username ] arg               username for authentication
  -p [ --password ] arg               password for authentication
  --authenticationDatabase arg        user source (defaults to dbname)
  --authenticationMechanism arg       authentication mechanism
  --gssapiServiceName arg (=mongodb)  Service name to use when authenticating 
                                      using GSSAPI/Kerberos
  --gssapiHostName arg                Remote host name to use for purpose of 
                                      GSSAPI/Kerberos authentication
```

Comment (caosiyang) [inline README.md]: Is "authdb" ok?

BTW:
The name "authenticationDatabase" refers to "mongo" command options.
"mongo --help" output like this:
```
Authentication Options:
  -u [ --username ] arg               username for authentication
  -p [ --password ] arg               password for authentication
  --authenticationDatabase arg        user source (defaults to dbname)
  --authenticationMechanism arg       authentication mechanism
  --gssapiServiceName arg (=mongodb)  Service name to use when authenticating 
                                      using GSSAPI/Kerberos
  --gssapiHostName arg                Remote host name to use for purpose of 
                                      GSSAPI/Kerberos authentication
```

Comment (siddontang) [reply README.md]: Oh......
make sense.

Comment (siddontang) [reply README.md]: here I just worry that some can not typo `authenticationDatabase` correctly.

Comment (caosiyang) [reply README.md]: Done. 
authenticationDatabase => **authdb**

**Pull request merged**